common/app/views/fragments/collections/popular.scala.html

```diff
@@ -7,76 +7,42 @@
 @import views.support.MostPopular.{showMPU, tabsPaneCssClass}
 
 @defining(popular.size > 1){ isTabbed =>
-    <div id="popular-trails" class="fc-slice fc-slice--popular" data-link-name="most popular @if(isMostPopularDefaultTest){ most-popular-default-ab-2}" data-test-id="popular-in">
+    <div id="popular-trails" class="fc-slice fc-slice--popular" data-link-name="most popular Test" data-test-id="popular-in">
         @if(isTabbed) {
             <div class="tabs u-cf">
                 <ol class="tabs__container js-tabs" id="js-popular-tabs" role="tablist">
-                    @if(isMostPopularDefaultTest) {
-                        @popular.reverse.zipWithRowInfo.map{ case (section, info) =>
-                        <li class="tabs__tab@if(info.isFirst){ tabs__tab--selected tone-colour tone-accent-border}" role="tab" id="tabs-popular-@info.rowNum-tab"@if(info.isFirst){ aria-selected="true"} aria-controls="tabs-popular-@info.rowNum">
-                            <a href="#tabs-popular-@info.rowNum" data-link-name="tab @info.rowNum @section.heading"><span class="u-h">Popular </span>@Html(Localisation(section.heading.stripPrefix("popular ")))</a>
-                        </li>
-                        }
-                    } else {
-                        @popular.zipWithRowInfo.map{ case (section, info) =>
-                        <li class="tabs__tab@if(info.isFirst){ tabs__tab--selected tone-colour tone-accent-border}" role="tab" id="tabs-popular-@info.rowNum-tab"@if(info.isFirst){ aria-selected="true"} aria-controls="tabs-popular-@info.rowNum">
-                            <a href="#tabs-popular-@info.rowNum" data-link-name="tab @info.rowNum @section.heading"><span class="u-h">Popular </span>@Html(Localisation(section.heading.stripPrefix("popular ")))</a>
-                        </li>
-                        }
+                    @popular.zipWithRowInfo.map{ case (section, info) =>
```

Comment:
Hey thanks for checking this! I thought this was a better way of doing it: https://github.com/guardian/frontend/blob/remove-pop-test/onward/app/controllers/MostPopularController.scala#L34